<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel de Control</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Arduino</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="errorArduinoMessageLabel">
         <property name="text">
          <string>Error message:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="arduinoTextBrowser"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="abrirArduinoPushButton">
           <property name="text">
            <string>Abrir Arduino</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="encenderPushButton">
           <property name="text">
            <string>Encender</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="cerrarArduinoPushButton">
           <property name="text">
            <string>Cerrar Arduino</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="apagarPushButton">
           <property name="text">
            <string>Apagar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Android</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="errorAndroidMessageLabel">
            <property name="text">
             <string>Error message:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Puerto:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="portSpinBox"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextBrowser" name="androidTextBrowser"/>
          </item>
          <item>
           <widget class="QPushButton" name="abrirServidorPushButton">
            <property name="text">
             <string>Abrir Servidor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cerrarServidorPushButton">
            <property name="text">
             <string>Cerrar Servidor</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
